apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var ConversationAccessor = require('./accessors');
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var requireFromApp = require(appRoot + '/shared/test/integration/require_from_app');
+
+function Conversation(client) {
+  this.client = client;
+  this.accessors = ConversationAccessor;
```

Comment:
It'll make the magical hack cleaner. Just one question, what's the difference between

```js
function MyObject(client) { this.client = client; }
MyObject.prototype.myFunction = function() { /* uses this.client */ }

module.exports = MyObject;
```

and

```js
module.exports = {
  create: function(client) {
    return {
      myFunction: function() { /* uses client */ }
    }
  }
}
```